{"name": "test_third", "status": "failed", "statusDetails": {"message": "AssertionError: assert 20 == 10", "trace": "def test_third():\n        a = 10\n        b = 20\n    \n>       assert b == a\nE       assert 20 == 10\n\nSeleniumProgram\\test_first_program.py:19: AssertionError"}, "start": 1672762896415, "stop": 1672762896415, "uuid": "71dbfd9f-6b88-4ed8-8251-c242a2ca8a1f", "historyId": "67567a4ffb3647a2a1fbe8cd079035b2", "testCaseId": "036ce87eb7f9a823a3348e34ea37842e", "fullName": "test_first_program#test_third", "labels": [{"name": "suite", "value": "test_first_program"}, {"name": "host", "value": "HP"}, {"name": "thread", "value": "24940-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_first_program"}]}